packaging/ubuntu-16.04/changelog

```diff
@@ -1,7 +1,337 @@
+      for process-control
+    - interfaces: add read access to /proc/cgroups and
+      /proc/sys/vm/swappiness to system-observe
+    - fde: run fde-reveal-key with `DefaultDependencies=no`
+    - many: don't concatenate non-constant format strings
+    - o/devicestate: fix non-compiling test
+    - release, snapd-apparmor: fixed outdated WSL detection
+    - many: add todos discussed in the review in
+      tests/nested/manual/fde-on-classic, snapstate cleanups
+    - overlord: run install-device hook during factory reset
+    - i/b/mount-control: add optional `/` to umount rules
+    - gadget/install: split Run in several functions
+    - o/devicestate: refactor some methods as preparation for install
+      steps implementation
+    - tests: fix how snaps are cached in uc22
+    - tests/main/cgroup-tracking-failure: fix rare failure in Xenial and
+      Bionic
+    - many: make {Install,Initramfs}{{,Host},Writable}Dir a  function
+    - tests/nested/manual/core20: fix manual test after changes to
+      'tests.nested exec'
+    - tests: move the unit tests system to 22.04 in github actions
+      workflow
+    - tests: fix nested errors uc20
+    - boot: rewrite switch in SnapTypeParticipatesInBoot()
+    - gadget: refactor to allow usage from the installer
+    - overlord/devicestate: support for mounting ubuntu-save before the
+      install-device hook
+    - many: allow to install/update kernels/gadgets on classic with
+      modes
+    - tests: fix issues related to dbus session and localtime in uc18
+    - many: support home dirs located deeper under /home
+    - many: refactor tests to use explicit strings instead of
+      boot.Install{Initramfs,Host}{Writable,FDEData}Dir
+    - boot: add factory-reset cases for boot-flags
+    - tests: disable quota tests on arm devices using ubuntu core
+    - tests: fix unbound SPREAD_PATH variable on nested debug session
+    - overlord: start turning restart into a full state manager
+    - boot: apply boot logic also for classic with modes boot snaps
+    - tests: fix snap-env test on debug section when no var files were
+      created
+    - overlord,daemon: allow returning errors when requesting a restart
+    - interfaces: login-session-control: add further D-Bus interfaces
+    - snapdenv: added wsl to userAgent
+    - o/snapstate: support running multiple ops transactionally
+    - store: use typed valset keys in store package
+    - daemon: add `ensureStateSoon()` when calling systems POST api
+    - gadget: add rules for validating classic with modes gadget.yaml
+      files
+    - wrappers: journal namespaces did not honor journal.persistent
+    - many: stub devicestate.Install{Finish,SetupStorageEncryption}()
+    - sandbox/cgroup: don't check V1 cgroup if V2 is active
+    - seed: add support to load auto import assertion
+    - tests: fix preseed tests for arm systems
+    - include/lk: update LK recovery environment definition to include
+      devi…
```

Comment:
where is ... coming from?

Reply:
Meh, this was merged like this sadly (https://github.com/snapcore/snapd/pull/11128) :(